node/include/cocaine/detail/service/node/engine.hpp

```diff
@@ -17,13 +17,16 @@
 #include "cocaine/detail/service/node/slave/load.hpp"
 #include "cocaine/detail/service/node/stats.hpp"
 
+#include "node/pool_observer.hpp"
```

Comment:
I prefer large-commit-that-changes-everything at least in current situation - we have so many developers and actively developed features that it will likely break nothing